app/controllers/samples_controller.rb

```diff
@@ -495,15 +495,14 @@ def update
   # DELETE /samples/1
   # DELETE /samples/1.json
   def destroy
-    deletable = @sample.pipeline_runs.empty?
```

Comment:
I think there should still be a guard here, only a different one -- i.e. check if the pipeline_run is indeed failed. Probably not enough to have the condition only shallowly in the frontend

Reply:
good idea